### PR TITLE
Add first version of why carlhacks section

### PR DESCRIPTION
**Pull request by lord**
The copy still needs work! I'd love suggestions.

A preview:

screen shot 2015-03-08 at 9 29 04 pm

On mobile devices, it looks like

screen shot 2015-03-08 at 9 30 59 pm

### CODE REVIEW

**Comment by Kenadia** [comment]
Looks and sounds awesome!! My thoughts:
- Delete 'available' in the first bullet point
- Leave out the last bullet point until we can link to the 'about' page

**Comment by lord** [comment]
Version 2: 
screen shot 2015-03-08 at 10 24 41 pm

File: views/index.jade [hunk 1]
```diff
@@ -38,6 +38,17 @@ block page
...
     section.card.left.info
+      aside.sidebar
+        h1 Why CarlHacks?
+        ul
+          li Mentorship from professors and engineers
+          li Git and web development workshops for beginners
+          li Talks throughout the event for proficient hackers
+          li Priority registration option for women
+          li Hardware lab
+          li Bouldering cave and 440-acre arboretum to explore
```
Comment (Machyne) [inline views/index.jade]: Arb should be 880 acres, no?

**Comment by courtneyj** [comment]
I'd change "healthy meals" to "delicious meals." Also, our Arb is 880 acres.

**Comment by Machyne** [comment]
Didn't test, but you have a `left` in the media query and not normally. Did you try sizing page down and back up? Doesn't always work.

**Comment by lord** [comment]
@Machyne Hmm...sizing seems to work fine for me. Maybe you're thinking the media query is for small screens only? I did the sidebar "mobile first", so the stuff in the media query is only applied when the screen is large enough.

**Comment by lord** [comment]
v3

screen shot 2015-03-08 at 11 55 34 pm

**Comment by Machyne** [comment]
Just tested, works fine. LGTM!

**Comment by AdamCanady** [comment]
deploy!

Adam Canady | (319) 512-8823 | LinkedIn
http://www.linkedin.com/in/adamcanady | AdamCanady.com
http://adamcanady.com/

**Pull request merged**